setup/artifacts/github.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: single
-title:  "Configuring GitHub Webhooks"
+title:  "Configure a GitHub artifact account"
 sidebar:
   nav: setup
 ---
```

Comment:
Looks like the first line in the description just below is also a copy/paste error.